### PR TITLE
fix(seeds): better error messages

### PR DESCRIPTION
**Pull request by Bushuo**
This PR adds an interceptor to the axios response in case of an error, and maps the message to a more user friendly string.

## Basics

- [X] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [X] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [X] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [X] I added code comments, logging, and assertions as appropriate
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

File: frontend/src/config/index.ts [hunk 1]
```diff
@@ -1 +1,18 @@
...
+
+// Intercept the axios response to map errors to strings.
+axios.interceptors.response.use(
+  (r) => r,
+  (error) => {
+    console.error(error);
+
+    if (error instanceof Error) {
+      error.message = mapErrorToString(error);
+    }
```
Comment (Benjaminimal) [inline frontend/src/config/index.ts:12]: Do you think we really need this guard here?
What else could be passed to this error handler?
From the docs:
> Any status codes that falls outside the range of 2xx cause this function to trigger
https://axios-http.com/docs/interceptors

**Review by Benjaminimal — COMMENTED** [review]
Looks good but someone should run `prettier` over the whole frontend sub dir.
Reviews with formatting changes are a pain.

**Comment by markus2330** [comment]
> Looks good but someone should run prettier over the whole frontend sub dir.
> Reviews with formatting changes are a pain.

Absolutely, please propose in #98 how exactly we should do the reformatting (exact command, arguments, config).

The reformatting, however, should happen in an extra PR. Till then it is best to do as little reformatting as possible.

File: frontend/src/config/index.ts (hunk 1 above)
Comment (Bushuo) [reply frontend/src/config/index.ts:12]: Yes you're right :) I will remove the guard

**Comment by Bushuo** [comment]
I agree with both of you. Thanks for the feedback 🎉🎉

**Pull request merged**